static/app/gettingStartedDocs/javascript/angular.tsx

```diff
@@ -31,71 +32,176 @@ import {
 import {ProductSolution} from 'sentry/components/onboarding/productSelection';
 import {t, tct} from 'sentry/locale';
 
-type Params = DocsParams;
+export enum AppType {
+  STANDALONE = 'standalone',
+  MODULE = 'module',
+}
+
+const platformOptions = {
+  appType: {
+    label: t('App Type'),
+    defaultValue: AppType.STANDALONE,
+    items: [
+      {
+        label: 'Standalone',
+        value: AppType.STANDALONE,
+      },
+      {
+        label: 'Module',
+        value: AppType.MODULE,
+      },
+    ],
```

Comment:
@Lms24 what would be the proper label for it?
Can we add some postfix to `Standalone` and `Module` to make it clearer?

Reply:
With "postfix", do you mean a tootlip-like element? Then I'd add something like

- standalone: "Use this if you have an `app.config.ts` file"
- module: "Use this if you have an `app.module.ts` file"

Some other ideas for the button names:

1. App Config / App Module
2. Standalone Config / Module-based Config
3. Standalone Config / NGModule Config
4. App Config / NGModule Config
5. Standalone Conponent Config / NGModule-based Config

After thinking a bit about it, I kinda like 3 and 4. Makes it really clear that we're talking about Angular's `NGModule`s. WDYT?

Reply:
Or can we do something like

"I have an [app.config.ts]|[app.module.ts] file", so that the selection more or less becomes part of a sentence? Hope this makes sense 😅

Reply:
Was talking about changing the button label 👍 
Will go for 4 then. that is the clearest for me as someone who is not super familiar with Angular.